iron-selectable.html

```diff
@@ -264,7 +267,8 @@
     },
 
     _valueForItem: function(item) {
-      return item[this.attrForSelected] || item.getAttribute(this.attrForSelected);
+      return item[Polymer.CaseMap.dashToCamelCase(this.attrForSelected)] ||
```

Comment:
Would it be worth memoizing the camelCase version when attrForSelected changes (in _updateSelected)?